packages/next/package.json

```diff
@@ -36,9 +36,9 @@
   },
   "peerDependencies": {
     "@tanstack/react-query": "^4.3.8",
-    "@trpc/client": "^10.0.0-alpha.48",
-    "@trpc/react": "^10.0.0-alpha.48",
-    "@trpc/server": "^10.0.0-alpha.48",
+    "@trpc/client": "^10.0.0-proxy-beta.21",
+    "@trpc/react-query": "^10.0.0-proxy-beta.21",
+    "@trpc/server": "^10.0.0-proxy-beta.21",
```

Comment:
might cause issues since 21 doesn't exist yet 🤔 but for npm these needs to be updated